hasher-matcher-actioner/hmalib/models.py

```diff
@@ -281,3 +283,56 @@ def from_time_range(
             & Key("UpdatedAt").between(start_time, end_time),
             ProjectionExpression=cls.DEFAULT_PROJ_EXP,
         ).get("Items", [])
+    @staticmethod
+    def from_dynamodb_string(string: str) -> "Label":
+        return Label(*string.split(":"))
+
+
+@dataclass
+class MatchMessage(SNSMessage):
+    """
+    Captures a match that will need to be processed. Joins the dataset and the
+    pipeline together.
```

Comment:
Are dataset and pipeline defined somewhere? Worried they are super generic terms (especially pipeline)

Reply:
No they are not. I haven't updated this since we discussed. Completely open to changing the names of the variables as we get clearer on the terminology.